aws/logs_monitoring/log-sam-template.yaml

```diff
@@ -1,13 +1,15 @@
 AWSTemplateFormatVersion: '2010-09-09'
 Transform: AWS::Serverless-2016-10-31
-Description: Pushes logs from S3 and CloudWatch Logs to Datadog.
+Description: Pushes logs and metrics from AWS to Datadog.
 Resources:
   loglambdaddfunction:
     Type: 'AWS::Serverless::Function'
     Properties:
-      Description: Pushes logs from S3 and CloudWatch Logs to Datadog.
+      Description: Pushes logs and metrics from AWS to Datadog.
       Handler: lambda_function.lambda_handler
-      MemorySize: 128
+      MemorySize: 1024
```

Comment:
Updated the parameters to match the recommended values in the public datadog document.